bitnami/external-dns/templates/secret.yaml

```diff
@@ -25,6 +25,9 @@ data:
   {{- if eq .Values.provider "google" }}
   credentials.json: {{ .Values.google.serviceAccountKey | b64enc | quote }}
   {{- end }}
+  {{- if and (eq .Values.provider "hetzner") (not .Values.hetzner.secretName) }}
+  hetzner_token: {{ .Values.hetzner.token | b64enc | quote }}
```

Comment:
Could you please add validation to check the token is provided like in the rest of the cases? Check the _helpers.tpl

Reply:
Check was added :)